release/pipline/dags/istio_common_dag.py

```diff
@@ -220,8 +220,9 @@ def GenerateTestArgs(**kwargs):
     -u "{{ settings.MFEST_URL }}" \
     -t "{{ m_commit }}" -m "{{ settings.MFEST_FILE }}" \
     -a {{ settings.SVC_ACCT }}
+    BUILD_RET=$?
     echo "{{ m_commit }}" >> green_build_sha.txt
-    gsutil cp green_build_sha.txt gs://{{ settings.GCS_BUILD_PATH }}/green_build_sha.txt
+    gsutil cp green_build_sha.txt gs://{{ settings.GCS_BUILD_PATH }}/green_build_sha.txt && exit $BUILD_RET
```

Comment:
Can we move 

    echo "{{ m_commit }}" >> green_build_sha.txt
    gsutil cp green_build_sha.txt gs://{{ settings.GCS_BUILD_PATH }}/green_build_sha.txt

somewhere above before running the build, leave the BUILD_RET and exit alone but add a comment explaining that we really want the build exit code